scripts/commands/getmod.lua

```diff
@@ -0,0 +1,24 @@
+---------------------------------------------------------------------------------------------------
+-- func: getmod
+-- desc: gets a mod on the target/player
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 3,
+    parameters = "is"
+};
+
+function onTrigger(player, id, target)
+    local effectTarget = player;
```

Comment:
I did not make me intention clear when writing this as I had originally tested this on monster and not players.  I'll have an update that uses the cursor target instead of target.